### DISCUSSION TITLE
txid explorer

### DISCUSSION THREAD

**Discussion by Caligo91** [community/community#75608] [Programming Help]
### Body

안녕하세요, btc txid를 대량으로 검색해서 엑셀로 결과 값을 볼 수 있도록 만들고 싶은데, btc.com의 api를 사용하는 방법이나 코드 알려주시면 감사하겠습니다.
Hello, I would like to search btc txid in bulk and display the results in Excel. I would appreciate it if you could tell me how to use btc.com's API or code.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by elderguardian** [comment]
https://explorer.btc.com/btc/adapter?type=api-doc
